### PR TITLE
0.3.13 - fix #5 and #6

### LINKED ISSUE #5: Embed assets with "!(url)" force onebox properly, even when extension is missing.
From http://chat.stackoverflow.com/transcript/message/22453844#22453844

When a message [(This one for example)](http://chat.stackoverflow.com/transcript/message/22453533#22453533) is posted with the `!` force picture oneboxing option with no extension, the script does not pick it up.

### PR DESCRIPTION
**Pull request by somebody1234**
Also make it work when logged out

### CODE REVIEW

**Comment by somebody1234** [comment]
Note that you'll need to make all links point back to here again

File: expando.user.js [hunk 1]
```diff
@@ -173,6 +189,13 @@
                 }
                 var images = mutation.addedNodes[j].querySelectorAll('img');
                 for (var k = 0; k < images.length; k++) {
+                    (function (image) {
```
Comment (MadaraUchiha) [inline expando.user.js]: You should just move the whole thing to `for (const image of images) {` and get rid of the closure.

**Review by MadaraUchiha — CHANGES_REQUESTED** [review]

File: expando.user.js (hunk 1 above)
Comment (somebody1234) [reply expando.user.js]: @MadaraUchiha Hmm, that would work. But would that be supported in enough browsers?

Comment (MadaraUchiha) [reply expando.user.js]: Supported in latest chrome and latest firefox. The rest can go duck themselves.

Comment (somebody1234) [reply expando.user.js]: @MadaraUchiha Hmm, in that case I'm going to refactor (?) it if that's okay with you

File: expando.user.js [hunk 2]
```diff
@@ -43,54 +43,53 @@
             }
             req.setRequestHeader('X-Requested-With', 'XMLHttpRequest');
 
-            req.send(serialize(data));
+            req.send((function serialize(obj) {
```
Comment (MadaraUchiha) [inline expando.user.js:41]: You've inlined the function here, but didn't get rid of the original 20 or so lines up.

**Review by MadaraUchiha — CHANGES_REQUESTED** [review]

File: expando.user.js (hunk 2 above)
Comment (MadaraUchiha) [reply expando.user.js:41]: Other than that, good to go :)

**Pull request merged**